website/blog/2019-12-19-improve-presto-planner.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Improving presto planner for better push down and data federation 
+author: Yi He 
+authorURL: https://www.linkedin.com/in/yi-he-69aa9723/
+---
+Yi He, James Sun, Maria Basmanova, Rongrong Zhong, Jiexi Lin, Saksham Sachdev
```

Comment:
This needs a short introduction without the giant image that can appear in the blog index. Put a 
`<!--truncate-->`
after the introduction.